static/app/views/performance/transactionSummary/transactionSpans/spanDetails/exclusiveTimeHistogram.tsx

```diff
@@ -145,7 +146,8 @@ export function Chart(props: ChartProps) {
       type: 'value' as const,
       axisLabel: {
         color: theme.chartLabel,
-        formatter: (value: number) => axisLabelFormatter(value, 'number'),
+        formatter: (value: number) =>
+          axisLabelFormatter(value, aggregateOutputType('number')),
```

Comment:
This also looks strange, `"number"` _is_ an aggregate output type. Should this be just `'number'`?

Reply:
Right, good catch that is strange. I imagine that just returns number. I will confirm!

Reply:
That's correct, it just returns number, updated!

Reply:
Wait hang on, I thought this can be changed to `axisLabelFormatter(value, 'number')` since `aggregateOutputType('number')` return `'number'`, no?